code/modules/clothing/suits/ego_gear/teth.dm

```diff
@@ -25,7 +27,7 @@ Any attempt to code risk class armor will result in a 10 day Github ban.*/
 	desc = "The archetype was already charred from the moment of extraction. \
 	Although the exterior is scorched, it has no adverse effects on the E.G.O’s performance."
 	icon_state = "match"
-	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = -20, BLACK_DAMAGE = -20, PALE_DAMAGE = 0) // 20
+	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = -20, BLACK_DAMAGE = -20, PALE_DAMAGE = 0, FIRE = 30) // 20
```

Comment:
If going by previous logic shouldnt the Match have higher resistance than its red? By default this would be 20 fire resistance but lifetime stew gets double its red resistance? Unless physical resistance and fire resistance are unrelated?

Shouldnt all fire resistant armor be equal in its defense against fire since its a out of context issue. I know you dont want people fighting abnormalities with teth armor but it feels weird having one fire abnormality just be weaker than another.

Reply:
It could be equal, but the idea is to give a little bit more to armors with worse spreads.